components/slider/stories/slider.stories.js

```diff
@@ -136,63 +158,94 @@ export default {
 	},
 };
 
+/**
+ * Sliders should always have a label. In rare cases where context is sufficient and an accessibility expert has reviewed the design, the label could be undefined. These sliders should still include an aria-label in HTML (depending on the context, “aria-label” or “aria-labelledby”).
```

Comment:
What do you think about taking this last sentence out, since it's mostly repeated below in the WithoutLabel story?

```suggestion
 * Sliders should always have a label. In rare cases where context is sufficient and an accessibility expert has reviewed the design, the label could be undefined.
```